coordinator/pom.xml

```diff
@@ -63,9 +63,9 @@
     <dropwizard.metrics.version>4.1.36</dropwizard.metrics.version>
     <!--
       Note that this is currently aligned with the Jetty version that DropWizard transitively depends on.
-      It's probably a good idea to keep the two versions in sync.
+      It's probably a good idea to keep the two versions in sync (or close)
     -->
-    <jetty.version>9.4.50.v20221201</jetty.version>
+    <jetty.version>9.4.51.v20230217</jetty.version>
```

Comment:
I would actually also like to see Dropwizard being bumped to `2.1.16`, which actually uses this version.. That being said, we could then invest time in bumping the dependencies across the project.. For example, logback could be bumped one version, maybe something else as well..

Reply:
I can try to see if that works fine, sure.

Reply:
@ivansenic Unfortunately it's not plug-in replacement without changes. I could spend more time figuring it out, but we are trying to get rid of DropWizard so for this change there's not much benefit.

Reply:
Ok yes I will file an issue.